buzzmobile/launch/simulation.launch

```diff
@@ -9,7 +9,7 @@
         <include file="$(find buzzmobile)/launch/includes/mapping.launch"/>
         <include file="$(find buzzmobile)/launch/includes/steering.launch"/>
         <include file="$(find buzzmobile)/launch/includes/lidar.launch"/>
-        <include file="$(find buzzmobile)/launch/includes/controller.launch"/>
+        <!--<include file="$(find buzzmobile)/launch/includes/controller.launch"/>-->
```

Comment:
Yeah I'm not sure why this worked either because one would think that the PID controller would need to poll the pose of the car to calculate output, but it worked. Nonetheless, I added the car_pose_mux import back.